test/compositemark/boxplot.test.ts

```diff
@@ -1555,6 +1556,29 @@ describe('normalizeBoxMinMax', () => {
       }
     );
   });
+
+  it("should not overwrite transform with boxplot's transfroms", () => {
+    const outputSpec = normalize(
+      {
+        data: {url: 'data/population.json'},
+        mark: {
+          type: 'boxplot',
+          extent: 'min-max'
+        },
+        transform: [{calculate: 'age * 2', as: 'age2'}],
+        encoding: {x: {field: 'age', type: 'ordinal'}, y: {field: 'people', type: 'quantitative', title: 'population'}}
+      },
+      defaultConfig
+    );
+
+    const transforms: Transform[] = outputSpec.transform;
+    if (transforms.length > 0) {
```

Comment:
Why add an if...else? Can you just add ad expect(transform).to.haveLengthAtLeast(1) or something?

Reply:
A failure in one assert should abort the test case.

Reply:
Ok, that seems a lot cleaner.

Reply:
We want to eventually change all assert statements but there is no point in doing that right now.